apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjFissPayer.java

```diff
@@ -75,50 +70,50 @@
   @Column(name = "`estAmtDue`", columnDefinition = "decimal(11,2)")
   private BigDecimal estAmtDue;
 
-  // BeneZ only
+  /** BeneZ only */
   @Column(name = "`beneRel`", length = 2)
   private String beneRel;
 
-  // BeneZ only
+  /** BeneZ only */
   @Column(name = "`beneLastName`", length = 15)
   private String beneLastName;
 
-  // BeneZ only
+  /** BeneZ only */
   @Column(name = "`beneFirstName`", length = 10)
   private String beneFirstName;
 
-  // BeneZ only
+  /** BeneZ only */
   @Column(name = "`beneMidInit`", length = 1)
   private String beneMidInit;
 
-  // BeneZ only
+  /** BeneZ only */
   @Column(name = "`beneSsnHic`", length = 19)
   private String beneSsnHic;
 
-  // Insured only
+  /** Insured only */
   @Column(name = "`insuredRel`", length = 2)
   private String insuredRel;
 
-  // Insured only
+  /** Insured only */
   @Column(name = "`insuredName`", length = 25)
   private String insuredName;
 
-  // Insured only
+  /** Insured only */
   @Column(name = "`insuredSsnHic`", length = 19)
   private String insuredSsnHic;
 
   @Column(name = "`insuredGroupName`", length = 17)
   private String insuredGroupName;
 
-  // Insured only
+  /** Insured only */
   @Column(name = "`insuredGroupNbr`", length = 20)
   private String insuredGroupNbr;
 
-  // BeneZ only
+  /** BeneZ only */
```

Comment:
❓ Wondering if there is a better way to indicate that some fields are only for some PayerTypes, in code. Maybe multiple classes that extend a base class? Not sure if that'd work with the implementation here, but just a thought.

Reply:
Good point.

Since we are using Hibernate and want to keep the implementation straight-forward this seemed like a good approach.  If we used multiple subclasses with different fields in each subclass we'd wind up having to do `instanceof` calls in code that queried the table.

Maybe we could map those fields to sub-entities and have hibernate populate them into fields like `getBeneZPayer()` and `getInsuredPayer()`.  That would be a whole new PR though since these fields pre-exist 0.5.

Reply:
Yup, I'd agree that would be work for another PR, but something to think about for the future 👍